docs/portal/api/removeEvent.md

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: 'removeEvent'
+sidebar_position: 4
+---
+# removeEvent
+
+
```

Comment:
Mention why and when we do this, it's when someone doesn't want to be reminded about the event, and click the don't remind me button (or something similar). I'm not sure about this so please bring it up in the next meeting